pallets/substrate-app/src/lib.rs

```diff
@@ -472,6 +485,40 @@ pub mod pallet {
             );
             Ok(())
         }
+
+        #[pallet::call_index(9)]
+        #[pallet::weight(<T as Config>::WeightInfo::register_erc20_asset())]
```

Comment:
You use `register_erc20_asset()` for weight everywhere, is it ok?
If you plan to replace it with associated weights, please add `TODO` comment

Reply:
it will be done in a separate task

Reply:
I think it's good to have `TODO` here as @wer1st suggested